configuration/globalproperties/autoCloseVisits.xml

```diff
@@ -0,0 +1,16 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>visits.autoCloseVisitType</property>
+      <value>
+	FIRST_CLINICAL_CONSULTATION,FIRST_APSS_CONSULTATION,FOLLOWUP_APSS_CONSULTATION,FOLLOWUP_CLINICAL_CONSULTATION
```

Comment:
@Edgencio since we can use the StaleVisitstask to remove old tasks I think we can remove this global property since this is only used by the AutoClose Task.

Reply:
Sure @nthfloor